includes/kvizzle_newdecls.inc

```diff
@@ -0,0 +1,1641 @@
+/*

```

Comment:
This is a fully newdecls version of kvizzle, however, it compiles with a billion warnings, due to the casting back and forth of `KeyValues` and `Handle` types, which is somewhat undefined

Reply:
I have a bunch of unit tests for Kvizzle.
With SM 1.10, both **kvizzle_newdecls** and **kvizzle** work fine.
With SM 1.11, only **kvizzle_newdecls** works.
That's fine for now - I'm using SM 1.10 for compilation. And when we switch to SM 1.11, we'll switch to kvizzle_newdecls.

Could the warnings be fixed with `view_as<>`?